app/src/main/java/com/duycuong/weather/ui/screen/main/MainActivity.java

```diff
@@ -0,0 +1,147 @@
+package com.duycuong.weather.ui.screen.main;
+
+import android.content.Intent;
+import android.databinding.BindingAdapter;
+import android.databinding.DataBindingUtil;
+import android.location.Location;
+import android.location.LocationListener;
+import android.location.LocationManager;
+import android.net.Uri;
+import android.os.Build;
+import android.provider.Settings;
+import android.support.annotation.NonNull;
+import android.support.design.widget.TabLayout;
+import android.support.v4.BuildConfig;
+import android.support.v4.view.ViewPager;
```

Comment:
ERROR: Unused import: android.support.v4.view.ViewPager. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports